### DISCUSSION TITLE
How to open new GUI window on current space?

### DISCUSSION THREAD

**Discussion by kiryph** [qvacua/vimr#947] [Q&A]
Let's say I have a running VimR instance with a GUI window on space 1.
When I switch to space 2, space 1 with the single VimR GUI window becomes hidden. Now I want to define a keyboard shortcut to create a new window on current space 2 without creating a new instance of VimR, e.g. with `$ open -n /Applications/VimR.app`.

The cli script from VimR `vimr` opens a new window on the same space as the existing window, i.e. on space 1 and a space change is triggered.

In contrast, the cli script from MacVim `mvim` opens a new GUI window on the current space.

Could the default behaviour be  changed?

Also AppleScript support could be added:

```
tell application "VimR"
    make new window
    activate
end tell
```

**Comment by qvacua** [comment]
`vimr -n file1 file2` will open the files in a new window.

**Reply by kiryph** [reply]
Yes, it opens the files in a new window. But the question is on how to open a new window on the current space when there are already open vimr windows on other spaces. 

`$ vimr -n` opens a new window on the space of an existing vimr window as mentioned in the question.
